test/models/work_test.rb

```diff
@@ -0,0 +1,74 @@
+require "test_helper"
+
+describe Work do
+  describe "validations" do
+    it "validates that it has a title" do
+      media = Work.first
+      media.title = nil
+
+      expect(media.valid?).must_equal false
+      expect(media.errors.messages).must_include :title
+      expect(media.errors.messages[:title]).must_equal ["can't be blank"]
+    end
+
+    it "validates that it has a category" do
+      media = Work.first
+      media.category = nil
+
+      expect(media.valid?).must_equal false
+      expect(media.errors.messages).must_include :category
+      expect(media.errors.messages[:category]).must_equal ["can't be blank"]
+    end
+
+    it "validates that a work must be unique" do
+      media = works(:treat)
+      media2 = Work.create(title: media.title)
+
+      expect(media2.valid?).must_equal false
+      expect(media2.errors.messages[:title]).must_equal ["has already been taken"]
+    end
+  end
+
+  describe "relationships" do
+    it "can have many votes" do
+      media = works(:treat)
+      user = User.create(username: "Hugo")
+      user_two = User.create(username: "Brian")
+      # expect(media.votes.count).must_equal 0
+      #
+      vote = Vote.create(work_id: media, user_id: user)
+      vote2 = Vote.create(work_id:  media, user_id: user_two)
```

Comment:
```suggestion
      vote = Vote.create(work: media, user: user)
      vote2 = Vote.create(work:  media, user: user_two)
```
This test is failing because the work and user aren't successfully getting tied to these votes. This is because when you use `work_id`, ActiveRecord actually wants an id. You can instead change the code to what I have above.

Reply:
```suggestion
      vote = Vote.create(work_id: media.id, user_id: user.id)
      vote2 = Vote.create(work_id:  media.id, user_id: user_two.id)
```
You may have seen this before. This would also work.